.jscsrc

```diff
@@ -1,8 +1,6 @@
 {
   "preset": "google",
   "disallowSpacesInAnonymousFunctionExpression": null,
-  "excludeFiles": [
-    "node_modules/**",
-    "gulpfile.babel.js"
-  ]
+  "esnext": true,
+  "excludeFiles": ["node_modules/**"]
```

Comment:
@nicolasgarnier can we update to drop the excludeFiles line here as `node_modules` are excluded by default per @hzoo?